### PR TITLE
allow configuring sushy-tools to ignore boot devices

### PR DESCRIPTION
**Pull request by osherdp**
Just a simple configuration that I'm missing in dev-scripts.

### CODE REVIEW

File: config_example.sh [hunk 1]
```diff
@@ -131,7 +131,10 @@ set -x
 #export PROVISIONING_NETWORK=fd00:1101::0/64
 
 # Provisioning network profile, can be set to "Managed" or "Disabled"
-#PROVISIONING_NETWORK_PROFILE=Managed
+#export PROVISIONING_NETWORK_PROFILE=Managed
+
+# Configure sushy-tools to allow booting from different devices
+#export SUSHY_IGNORE_BOOT_DEVICE=False
```
Comment (dtantsur) [inline config_example.sh:137]: Sushy is a codename that not everyone knows. Let's rename the option and rephrase the comment in terms of a redfish emulator.

Comment (osherdp) [reply config_example.sh]: agreed, have another look

**Review by elfosardo — APPROVED** [review]

**Review by flaper87 — APPROVED** [review]
/lgtm

**Pull request merged**